nautobot/wireless/factory.py

```diff
@@ -0,0 +1,115 @@
+import factory
+
+from nautobot.core.factory import (
+    BaseModelFactory,
+    get_random_instances,
+    NautobotBoolIterator,
+    PrimaryModelFactory,
+    random_instance,
+    UniqueFaker,
+)
+from nautobot.dcim.models import Controller, Device
+from nautobot.ipam.models import VLAN
+from nautobot.tenancy.models import Tenant
+from nautobot.wireless import models
+from nautobot.wireless.choices import (
+    ChannelWidthChoices,
+    RadioFrequencyChoices,
+    RadioStandardChoices,
+    RegulatoryDomainChoices,
+    WirelessAuthTypeChoices,
+    WirelessDeploymentModeChoices,
+)
+
+
+class AccessPointGroupFactory(PrimaryModelFactory):
+    class Meta:
+        model = models.AccessPointGroup
+        exclude = ("has_description", "has_tenant")
+
+    name = UniqueFaker("company")
+    has_description = NautobotBoolIterator()
+    description = factory.Maybe("has_description", factory.Faker("sentence"), "")
+    controller = random_instance(Controller, allow_null=True)
+    has_tenant = NautobotBoolIterator()
+    tenant = factory.Maybe("has_tenant", random_instance(Tenant), None)
+
+
+class SupportedDataRateFactory(PrimaryModelFactory):
+    class Meta:
+        model = models.SupportedDataRate
+
+    standard = factory.Faker("random_element", elements=RadioStandardChoices.values())
+    rate = factory.Faker("pyint", min_value=1000, max_value=164000, step=1000)
+    mcs_index = factory.Faker("pyint", min_value=0, max_value=9)
+
+
+class RadioProfileFactory(PrimaryModelFactory):
+    class Meta:
+        model = models.RadioProfile
+        exclude = ("has_description",)
+
+    name = UniqueFaker("word")
+    frequency = factory.Faker("random_element", elements=RadioFrequencyChoices.values())
+    channel_width = factory.Faker("random_elements", elements=ChannelWidthChoices.values())
+    regulatory_domain = factory.Faker("random_element", elements=RegulatoryDomainChoices.values())
+    tx_power_min = factory.Faker("pyint", min_value=1, max_value=5)
+    tx_power_max = factory.Faker("pyint", min_value=6, max_value=30)
+    rx_power_min = factory.Faker("pyint", min_value=1, max_value=10)
```

Comment:
Nitpick: Are negative values ever used for these?

Reply:
I am not sure. Cisco is 1-8, HPE is 1-5 (maybe, some docs show to use DBm, others show a mapping between a number and the DBm), Android supports negative numbers. The model itself is IntegerField instead of positive integer field. For test data, it doesn't really matter, I just wanted to make sure that the tx_power_min would not be higher than the tx_power_max.